src/sitemap/sitemaps.py

```diff
@@ -214,3 +220,48 @@ def __init__(self, region, language):
         self.queryset = self.queryset.filter(
             poi__in=self.region.pois.all(), language=self.language
         )
+
+
+class OfferSitemap(WebappSitemap):
+    """
+    This sitemap contains all urls to offers for a specific region.
+
+    :param priority: The priority of this sitemap's urls (``1.0``)
+    :type priority: float
+
+    :param queryset: The queryset of this sitemap
+    :type queryset: :class:`~django.db.models.query.QuerySet` [ :class:`~cms.models.offers.offer.Offer` ]
+
+    Parameters inherited from :class:`~sitemap.sitemaps.WebappSitemap`:
+
+    :param changefreq: The usual change frequency of this sitemap's urls (``monthly``)
+    :type changefreq: str
+    """
+
+    priority = 1.0
+    queryset = Offer.objects.all()
+
+    def __init__(self, region, language):
+        """
+        This init function filters the queryset of offers objects based on the given region.
+
+        :param region: The region of this sitemap's urls
+        :type region: ~cms.models.regions.region.Region
+
+        :param language: The language of this sitemap's urls
+        :type language: ~cms.models.languages.language.Language
+        """
+        # Instantiate WebappSitemap
+        super().__init__(region, language)
+        # Filter queryset based on region
+        self.queryset = self.queryset.filter(region=self.region)
+        self.language = language
```

Comment:
```suggestion
```
You can omit this line, since the language attribute already gets assigned in the `__init__()`-method of the [super class](https://github.com/Integreat/cms-django/blob/d905ab25d8425c829972e70da1e3b9f25ea61536/src/sitemap/sitemaps.py#L48).